docker-compose.yml

```diff
@@ -94,6 +109,26 @@ services:
     depends_on:
       - flask
 
+  metabase-app:
```

Comment:
Ce qui me gêne un petit peu, c'est que ça alourdit notre recette alors que ces deux containers ne vont pas être utilisé lors de nos développements.
On pourrait créer une seconde recette comme je l'ai fait sur [mon dépôt](https://github.com/NerOcrO/docker-off).
Et après on peut faire un [enchaînement de recette](https://github.com/NerOcrO/docker-off/tree/master/docker#start-the-containers-with-elasticsearchlogstashkibana).
Sinon (je n'ai pas cherché), s'il y moyen de lancer docker-compose en sans certains container, je suis preneur aussi.

Reply:
On peut placer ces containers dans un autre fichier et le append à l'autre avec un -f  
[La doc :](https://docs.docker.com/compose/reference/overview/)
You can supply multiple -f configuration files. When you supply multiple files, Compose combines them into a single configuration. Compose builds the configuration in the order you supply the files. Subsequent files override and add to their predecessors.

Mais du coup il faudrait modifier la commande pc start backend;

Reply:
Je suis pour avoir 2 commandes séparées. Je ne vois pas de scénario pour lequel on aurait besoin de lancer le tout